src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj

```diff
@@ -30,5 +30,4 @@
         <ProjectReference Include="..\..\..\Bit.Client.Web.BlazorUI\Bit.Client.Web.BlazorUI.csproj" />
         <ProjectReference Include="..\Shared\Bit.Client.Web.BlazorUI.Playground.Shared.csproj" />
     </ItemGroup>
-
```

Comment:
please revert this change too.
there shouldn't be any changes in the project file in this PR